model-optimizer/mo/main.py

```diff
@@ -404,4 +435,5 @@ def main(cli_parser: argparse.ArgumentParser, framework: str):
 
 if __name__ == "__main__":
     from mo.utils.cli_parser import get_all_cli_parser
-    sys.exit(main(get_all_cli_parser(), None))
+    fem = FrontEndManager()
+    sys.exit(main(get_all_cli_parser(fem), fem, None))
```

Comment:
I think we can `FrontEndManager` inside main for any frontend type as it is always available.

Reply:
The reason why FrontEndManager is created here is that we need to use only one instance to avoid duplication of searching/loading frontends.
And FrontEndManager is needed in 'get_all_cli_parser' (to check available frontends) and also in 'main'.
My initial implementation created and returned FrontEndManager inside 'get_all_cli_parser', but it affected a lot of tests which used MO-functions directly. So I found that it is easier to create FrontEndManager inside __main__ and pass it to get_all_cli_parser and 'main'